lib/registration.pm

```diff
@@ -75,8 +77,26 @@ our %SLE15_DEFAULT_MODULES = (
     sles4sap => 'base,desktop,serverapp,ha,sapapp',
 );
 
+our %ADDONS_REGCODE = (
+    'sle-ha'                   => get_var('SCC_REGCODE_HA'),
+    'sle-ha-geo'               => get_var('SCC_REGCODE_GEO'),
+    'sle-we'                   => get_var('SCC_REGCODE_WE'),
+    'sle-module-live-patching' => get_var('SCC_REGCODE_LIVE'),
+    'sle-live-patching'        => get_var('SCC_REGCODE_LIVE'),
+    'SLES-LTSS'                => get_var('SCC_REGCODE_LTSS'),
+    'SUSE-Linux-Enterprise-RT' => get_var('SCC_REGCODE_RT'),
+);
+
 our @SLE15_ADDONS_WITHOUT_LICENSE        = qw(ha sdk wsm we hpcm live);
 our @SLE15_ADDONS_WITH_LICENSE_NOINSTALL = qw(ha we);
+our @SLE12_MODULES                       = qw(
```

Comment:
Please add comments to describe this table's using, or people will confuse why not all SLE12 modules listed here.